services/api-server/Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG PYTHON_VERSION="3.6.10"
-FROM python:${PYTHON_VERSION}-slim as base
+FROM python:${PYTHON_VERSION}-slim-buster as base
```

Comment:
buster is to reference a specific version of debian? without it it changes faster?

Reply:
yes, the idea is to pin it down to the maximum. Since we moved to slim it already moved from stretch to buster. see https://hub.docker.com/_/python